validator/src/main/resources/validations/otlp-mocked-server-prometheus-validation.yml

```diff
@@ -0,0 +1,5 @@
+-
```

Comment:
This looks just like the `default-mocked-server-validation.yml`

Reply:
Yes it is the same since mock server tests do not validate data, it only validates if data is sent to the mock server do you think I should use that yml file?

Reply:
What's the difference between these files? https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/validator/src/main/resources/validations/default-mocked-server-validation.yml

Reply:
If they're the same, then I don't see a reason to make another one.